src/spectral/rulesets/.defaultsForSpectral.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Was there some reason to avoid using yaml?

Reply:
None at all, just my personal preference. I always found it easier to configure a json file vs a yaml. I just took this phrase from their documentation literally:
_A ruleset is a JSON or YAML file (often the file will be called .spectral.yaml), and there are two main parts_
I am completely not against changing our default ruleset to yaml! I'll go ahead and make that change